frontend/javascripts/oxalis/view/action-bar/save_button.js

```diff
@@ -107,11 +125,19 @@ class SaveButton extends React.PureComponent<Props, State> {
         className={this.props.className}
         style={{ background: showUnsavedWarning ? "var(--ant-error)" : null }}
       >
-        {this.shouldShowProgress() ? (
-          <span style={{ marginLeft: 8 }}>{Math.floor((progressFraction || 0) * 100)} %</span>
-        ) : (
-          <span className="hide-on-small-screen">Save</span>
-        )}
+        <Tooltip
+          title={
+            totalBucketsToCompress > 0
+              ? `${totalBucketsToCompress} buckets remaining to compress...`
```

Comment:
Ideally, this tooltip wouldn't use the word "bucket", since we cannot expect the user to know this term. However, I don't have a better idea. One could convert to `MB`, but I'm not really fond of this idea.

Also cc @fm3 :)

Reply:
You could simply refer to them as items. This would be rather vague but still get the message across that things are happening (or need to happen) :)